wiki/Help_Centre/en.md

```diff
@@ -778,36 +828,40 @@ Make sure you provide an email address you have control over. This is used to re
 
 To upload your beatmap, go into the editor and hit `File` > `Upload Beatmap`.
 
-This will make your beatmap available for others to download on the website and create a dedicated thread for discussion. Updating a map is the same process for a map that has already been uploaded.
+This will make your beatmap available for others to download on the website. Updating a map is the same process for a map that has already been uploaded.
 
-Each user has a limited number of maps they can simultaneously upload. Leaving a map inactive for a month (by not updating it or not posting in its thread) will send it to the Beatmap Graveyard. To update a graveyarded beatmap, you need to revive it first.
+Each user has a limited number of maps they can simultaneously upload. Leaving a map inactive for a month (28 days), by not updating it, will send it to the [Beatmap Graveyard](/wiki/Graveyard). To revive a beatmap from the graveyard back into a Pending or WIP status, all you have to do is update it.
 
-##### The editor is telling me that beatmap you are trying to submit isn't yours!
+##### The editor is telling me the beatmap you are trying to submit isn't yours!
```

Comment:
```suggestion
##### The editor is telling me, "The beatmap you are trying to submit isn't yours!"
```

Reply:
Thank you, ill fix this up tonight and have another read through it myself